website/docs/cloud-docs/users-teams-organizations/organizations.mdx

```diff
@@ -44,7 +44,7 @@ On Terraform Cloud, any user can create a new organization. If you do not belong
 
 1. Click the current organization name and select **Create new organization**. The **Create a new organization** page appears.
 1. Enter a unique **Organization name** Organization names can include numbers, letters, underscores (`_`), and hyphens (`-`).
-1. Provide an **Email address** where you will receive notifictions about the organization.
+1. Provide an **Email address** where you will receive notifications about the organization.
```

Comment:
```suggestion
1. Provide an **Email address** to receive notifications about the organization.
```

Suggest changing "where you will receive" to "to receive" to use fewer words and avoid using future tense.